README.md

```diff
@@ -123,11 +123,14 @@ Many Jupyter kernels for functional programming languages are available such as
 | Stdin                  | Yes           | No            |
 
 In addition, the installer of OCaml Jupyter automatically adds the kernel to Jupyter.
+[simple_jucaml][simple_jucaml] and [jupyter-kernel][jupyter-kernel], other kernels for OCaml, seem too simple to use in practice.
```

Comment:
Careful, [jupyter-kernel] is not a Jupyter kernel for OCaml,
it's a library for *writing* Jupyter kernels (in OCaml). See
https://github.com/ocaml-jupyter/jupyter-kernel/issues/6
https://github.com/ocaml-jupyter/jupyter-kernel/pull/7
I think it also deserves to be cited as related work,
but with the need to emphasize this.

Reply:
Thanks. I misunderstood.